core/src/test/scala/unit/kafka/server/FetchRequestDownConversionConfigTest.scala

```diff
@@ -222,9 +225,14 @@ class FetchRequestDownConversionConfigTest extends BaseRequestTest {
     } else {
       assertEquals(Errors.UNSUPPORTED_VERSION, error(partitionWithDownConversionDisabled))
     }
+
     TestUtils.waitUntilTrue(() => TestUtils.metersCount(BrokerTopicStats.FetchMessageConversionsPerSec) > initialFetchMessageConversionsPerSec,
       s"The `FetchMessageConversionsPerSec` metric count is not incremented after 5 seconds. " +
       s"init: $initialFetchMessageConversionsPerSec final: ${TestUtils.metersCount(BrokerTopicStats.FetchMessageConversionsPerSec)}", 5000)
+
+    TestUtils.waitUntilTrue(() => TestUtils.metersCount(fetchMessageConversionsTimeMsMetricName) > initialFetchMessageConversionsTimeMs,
+      s"The `MessageConversionsTimeMs` in fetch request metric count is not incremented after 5 seconds. " +
+      s"init: $initialFetchMessageConversionsTimeMs final: ${TestUtils.metersCount(fetchMessageConversionsTimeMsMetricName)}", 5000)
```

Comment:
Could you add tea for `TemporaryMemoryBytes`? It seems to me `TemporaryMemoryBytes` also get fixed by this patch .

Reply:
Good point. Updated.